WARNINGS.md

```diff
@@ -141,6 +141,34 @@ or at the beginning of a rule.
 
 --------------------------------------------------------------------------------
 
+## <a name="load"></a>Same label is used for multiple loads
```

Comment:
Please rename to `same_origin_load`

Reply:
sorry about that and thanks for the catch!